api/SVGAElement.json

```diff
@@ -195,7 +195,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "61"
+              "version_added": "52"
```

Comment:
The PR description says this change on SVGAElement isn't included, but here it is. Should it be?

Reply:
It wasn't supposed to be, no, whoops!